flamedisx/source.py

```diff
@@ -502,7 +574,7 @@ def random_truth(self, n_events, fix_truth=None, **params):
         """Draw random "deep truth" variables (energy, position) """
         assert isinstance(n_events, int), \
             f"n_events must be an int, not {type(n_events)}"
-        return pd.DataFrame({'energy': np.ones(n_events)})
+        return pd.DataFrame({'energy': np.ones(n_events) * 5.})
```

Comment:
Why 5.? Are there cases where we want something different?

Reply:
Yes, in all cases. Removed default and raised NotImplementedError.